go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -1469,7 +1469,7 @@ func (fbo *folderBranchOps) partialMarkAndSweepLoop(trigger <-chan struct{}) {
 			continue
 		case _, ok := <-trigger:
 			if !ok {
-				fbo.log.CDebugf(ctx, "Mark-and-sweep is shutting down.")
+				fbo.log.CDebugf(ctx, "Mark-and-sweep is shutting down")
```

Comment:
lol why

Reply:
Just cause I noticed it.  It's a convention not to end log lines in punctuation.